app/app.js

```diff
@@ -150,6 +150,7 @@ angular.module('app', ['angular-md5'])
 
     if (currentCollectionIndexInCollections > 0) {
       prevCollection = collections[currentCollectionIndexInCollections - 1];
+      vm.prevCollection = prevCollection;
```

Comment:
Instead of assigning it like this, we can just always use `vm.prevCollection` in this file